BookShelfBusinessLogic/BookView.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace BookShelfBusinessLogic
+{
+    /// <summary>
+    /// Represents an entity for user-friendly representation of specific book
+    /// </summary>
+    public class BookView
+    {
+        /// <summary>
+        /// Initializes new instance of BookView class
```

Comment:
You can use this tag and attribute in documentation here<see cref="BookView"/> .